be/tool/beck.c

```diff
@@ -1566,6 +1595,187 @@ static void builder_fini(struct builder *b)
 	       b->b_act, b->b_tx);
 }
 
+static void be_cfg_update(struct m0_be_domain_cfg *cfg,
+			  const char              *str_key,
+			  const char              *str_value)
+{
+	uint64_t  value1_64;
+	uint64_t  value2_64;
+	uint64_t  value1_32;
+	char     *s1;
+	char     *s2;
+	bool      value_overridden = true;
+
+	/** Cover variables accepting two comma-separated values. */
+	if (m0_streq(str_key, "tgc_size_max")  || 
+	    m0_streq(str_key, "bec_tx_size_max")) {
+
+		s1 = m0_strdup(str_value);
+		s2 = strchr(s1, ',');
+
+		*s2 = '\0';
+		s2++;
+
+		value1_64 = m0_strtou64(s1, NULL, 10);
+		value2_64 = m0_strtou64(s2, NULL, 10);
+
+		m0_free(s1);
+
+		M0_ASSERT_INFO((value1_64 != ULLONG_MAX) &&
+				(value2_64 != ULLONG_MAX),
+				"Invalid value %s for variable %s in yaml file.", str_value, str_key);
+
+		if (m0_streq(str_key, "tgc_size_max")) {
+			cfg->bc_engine.bec_group_cfg.tgc_size_max = 
+					M0_BE_TX_CREDIT(value1_64, value2_64);
+		}
```

Comment:
[Style] keep both brace and else on same line, here and below as well.
} else {

Reply:
Done.